tools/internal_ci/linux/grpc_python_bazel_test_in_docker.sh

```diff
@@ -29,3 +29,6 @@ bazel test --spawn_strategy=standalone --genrule_strategy=standalone --test_outp
 bazel clean --expunge
 bazel test --config=python3 --spawn_strategy=standalone --genrule_strategy=standalone --test_output=errors //src/python/...
 bazel test --config=python3 --spawn_strategy=standalone --genrule_strategy=standalone --test_output=errors //examples/python/...
+(cd /var/local/git/grpc/bazel/test/python_test_repo;
```

Comment:
You could make an argument in either direction. Honestly, it would be preferable for Fusion to have knowledge of each of our tests at the test case level, but its support for test detection isn't great. Considering that this change wouldn't be API breaking and it will require adding a new job within the google3 kokoro configuration, I've created [an issue](https://github.com/grpc/grpc/issues/19854) to track doing this work in a separate PR.

Reply:
TODO sounds good, can you add a comment?
If this case has flake in the future, it is hard to determine if it is Bazel's fault or there's a bug. Separation provides better developer experience.